core/src/com/unciv/logic/GameStarter.kt

```diff
@@ -428,9 +428,14 @@ object GameStarter {
 
             //Trigger any global or nation uniques that should triggered.
             //We may need the starting location for some uniques, which is why we're doing it now
-            for (unique in ruleset.globalUniques.uniqueObjects + civ.nation.uniqueObjects)
+            val startingTriggers = (ruleset.globalUniques.uniqueObjects + civ.nation.uniqueObjects)
+                //We don't want uniques with trigger conditionals. Those are called when they are relevant
+                .filter { unique -> unique.conditionals.none {
+                    conditional -> conditional.type!!.targetTypes.any {
+                        it.canAcceptUniqueTarget(UniqueTarget.TriggerCondition) } } }
```

Comment:
I think there are several places in code where we make this check, should probably be a function on the unique class

Reply:
Sure I can do that. While I'm at it, I can fix the issue of unitTriggerConditions not being excluded in the areas where it previously did the check